lib/kitto/time.ex

```diff
@@ -13,7 +13,7 @@ defmodule Kitto.Time do
   """
   def mseconds({n, :milliseconds}), do: n
 
-  def mseconds({n, duration}) when duration in [:second, :minute, :hour, :day] do
+  def mseconds({_n, duration}) when duration in [:second, :minute, :hour, :day] do
```

Comment:
Well this should really be:

```def mseconds({1, duration}) when duration in [:second, :minute, :hour, :day] do
# something
end
```

It doesn't make sense to match when the user calls it like `mseconds({5, :second})`.